schema/types/isolator.go

```diff
@@ -69,8 +112,17 @@ func (is *Isolators) Unrecognized() Isolators {
 // serialized as any arbitrary JSON blob. Specific Isolator types should
 // implement this interface to facilitate unmarshalling and validation.
 type IsolatorValue interface {
+	// UnmarshalJSON unserialize a JSON-encoded isolator
 	UnmarshalJSON(b []byte) error
+	// AssertValid returns a non-nil error value if an IsolatorValue is not valid
+	// according to appc spec
 	AssertValid() error
+	// MaxInstances specifies the maximum number of allowed isolator instances
+	// for this isolator type
+	MaxInstances() uint
```

Comment:
MaxInstances always returns either `1` or `^uint(0)` (aka `math.MaxUint32`?). I don't see a use case for other values. Could it be `multipleAllowed() bool` instead?